README.md

```diff
@@ -11,14 +11,11 @@ The [elk](https://github.com/OpenKieler/elkjs) layout algorithm adapter for Cyto
 ## Dependencies
 
  * Cytoscape.js ^3.2.0
- * elkjs ^0.3.0
```

Comment:
Is the dependency bundled?  Does it need to be?  It would be great for users to be able to pull in elk updates without having the extension itself updated.

Reply:
Yes, as said in the PR description.

> Upgrade to Elk 0.7. As a side effect, this means all of Elk is now bundled into the browser-compatible dist/ file, so it's quite a bit larger than before. Closes #14. Refs #10, #13.

Reply:
Why would it need to be bundled?  The variations of elk files are the same as before: 

https://unpkg.com/browse/elkjs@0.7.0/lib/

Reply:
Reverted this change.